deploy.sh

```diff
@@ -14,6 +14,6 @@ else
     ssh-add deploy_key
     echo "$SERVER $SERVER_PUBLIC_KEY" > known_hosts
     echo "$NEW_SERVER $NEW_SERVER_PUBLIC_KEY" >> known_hosts
-    rsync --archive --verbose --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" src/ $DEPLOY_USER@$SERVER:whatwg.org/
-    rsync --archive --chmod=D755,F644 --verbose --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" idea.whatwg.org images.whatwg.org n.whatwg.org deploy@$NEW_SERVER:/var/www/
+    rsync --archive --verbose --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" whatwg.org/ $DEPLOY_USER@$SERVER:whatwg.org/
```

Comment:
Deliberately not simplifying this, trailing slashes in rsync make for fun mistakes.